web/settings/docker.py.example

```diff
@@ -1,12 +1,9 @@
-with open("/app/db/secret.txt", "r") as f:
+with open("/secret.txt", "r") as f:
     SECRET_KEY = f.readline()
 
 DATABASES = {
     "default": {
         "ENGINE": "django.db.backends.sqlite3",
-        "NAME": "/app/db/kipa.db",
+        "NAME": "/tmp/kipa.db",
```

Comment:
Laittaisin db:n kipan asennuskansioon, ottaen huomioon tmp-kansion epävakauden; db tuotantokäytössä pitäisi säilyä kaikissa tilanteissa

Reply:
Niin totta, en tosiaan tiiä miten /tmp-kansio käyttäytyy esim. kontin stopissa tai startissa.

Reply:
Tein tietokannalle konttiin ihan oman hakemiston. Lisäsin myös asennusohjeeseen kuinka sen saa bind mountilla jaettua hostin kanssa niin ei mene data hukkaan kontin sammuessa.

Reply:
Edelleen pitäisin oletuksena, että käyttäjä ei ole teknisesti vihkiytynyt ja näin ollen oletuksena kannan tulisi säilyä ilman erillistä conffausta.

Devaaminen toki asia erikseen, mutta tämä olisi korvaava asennustapa nykyiselle exelle ja näin ollen odottaisin vähintään yhtä hyvää käyttäjäkokemusta.

Mitä ajatuksia herättää?

Reply:
Voishan siihen ensimmäiseen dokumentaatiossa olevaan komentorimpsuun laittaa nimetyn volumen luonnin mukan: `--volume kipa-volume:/db`, jolloin se jäis henkiin vaikka kontin tappaisi ja käynnistäisi uudestaan.

Reply:
Liittyen tuohon huomioon vihkiytymättömistä käyttäjistä: Asennusdokkarissa vois olla oma kappaleensa Dockerista ja siitä, että miksi sitä käytetään. Ehkä jopa ohjaus ulkopuoliseen dokumentaatioon sen asentamisesta.